src/lecture/ten/numberSequence/Main.java

```diff
@@ -0,0 +1,26 @@
+package lecture.ten.numberSequence;
+
+import java.util.ArrayDeque;
+import java.util.Iterator;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        int i = 0;
+        Integer container;
```

Comment:
maybe better move "Integer container" in the while cycle

Reply:
This container is used in two different while cycles. If I move it to one of them, the other won't see it. I'd prefer just to use it as a global variable, is that okay?